src/platform/plugins/shared/unified_histogram/public/utils/external_vis_context.ts

```diff
@@ -104,9 +105,21 @@ export const isSuggestionShapeAndVisContextCompatible = (
   );
 };
 
+const injectIntervalToDateTimeColumn = (
+  columns: TextBasedLayerColumn[],
+  dateFieldLabel: string
+) => {
+  const dateColumn = columns.find((column) => column.columnId === 'timestamp');
```

Comment:
What if current data set has a real `timestamp` field, would not it conflict with this additional logic?

Reply:
These are the columns used by the histogram and not the entire dataset, so this is the column we create with the eval

Reply:
Alright! Can we create a const for `timestamp` and share between files?

Reply:
Here it is Jul https://github.com/elastic/kibana/pull/208053/commits/84cef9187606ce81903c8ef0027c190cbd020242